vrecord

```diff
@@ -1165,16 +1168,20 @@ MIDDLEOPTIONS+=(${EXTRAOUTPUTOPTIONS[@]})
 # set up input and playback
 _set_up_drawtext
 _get_decklink_inputs
```

Comment:
This needs to be moved under the `else` at 1173. After making that change this works on my mac laptop!

Reply:
(and by this I mean the `_get_decklink_inputs`